### PR TITLE
Approval email for newly registered users

### PR DESCRIPTION
**Pull request by yasmin2496**
## Purpose
_Trigger email for admin to approve new users_

### Further Info
Ticket number: #55
 
closing #55 

@RadhikaPPatel @vishalajackus

### CODE REVIEW

File: buildly/settings/base.py [hunk 1]
```diff
@@ -198,3 +198,12 @@
 SWAGGER_SETTINGS = {
     'DEFAULT_INFO': 'gateway.urls.swagger_info',
 }
+
+ORGANIZATION_TYPES = [
```
Comment (RadhikaPPatel) [inline buildly/settings/base.py:202]: Need to be Custodian and Producer (for now) as discussed on Friday meeting.

File: buildly/tests/test_loadinitialdata.py [hunk 2]
```diff
@@ -34,6 +34,7 @@ def test_full_initial_data(self):
         call_command('loadinitialdata', *args, **opts)
 
         assert CoreGroup.objects.filter(name='Global Admin', is_global=True, permissions=15).count() == 1
+        assert OrganizationType.objects.filter().count() >= 6
```
Comment (RadhikaPPatel) [inline buildly/tests/test_loadinitialdata.py:37]: Might wanna correct this as well as per the above changes in org types.

File: core/serializers.py [hunk 3]
```diff
@@ -138,6 +138,24 @@ def create(self, validated_data):
...
+        html_template_name = 'email/coreuser/approval.html'
+        context = {
+                    'approval_link': approval_link,
+                    'coreuser_name': coreuser.first_name + ' ' + coreuser.last_name,
+                    'organization_name': organization
+        }
+        if is_new_org:
+            admin = CoreUser.objects.filter(is_superuser=True).first()  # Global Admin
+        else:
+            org_admin_groups = CoreGroup.objects.filter(permissions=PERMISSIONS_ORG_ADMIN, is_org_level=True)
```
Comment (RadhikaPPatel) [inline core/serializers.py:154]: Might be multiple org admins. So email to all org admins.

**Review by RadhikaPPatel — CHANGES_REQUESTED** [review]
Please check the comments.

**Review by vishalajackus — APPROVED** [review]

**Pull request merged**